pkg/alertmanager/api.go

```diff
@@ -422,15 +422,9 @@ func validateReceiverHTTPConfig(cfg commoncfg.HTTPClientConfig) error {
 	if cfg.BearerTokenFile != "" {
 		return errPasswordFileNotAllowed
 	}
-	if cfg.ProxyURL.URL != nil {
-		return errProxyURLNotAllowed
```

Comment:
Are we still using `errProxyURLNotAllowed`? I would have expected that variable to be removed too.